core/src/fwd/Kokkos_Fwd_SYCL.hpp

```diff
@@ -49,7 +49,8 @@
 namespace Kokkos {
 namespace Experimental {
 class SYCLDeviceUSMSpace;  ///< Memory space on SYCL device
-class SYCL;                ///< Execution space for SYCL
+class SYCLSharedUSMSpace;
```

Comment:
Make sure you comment and clearly disambiguate it from `SYCLDeviceUSMSpace`.
Shared memory accessible by both host and device versus device memory not accessible by the host.